app/k8s/job/arbimon-rfm.json

```diff
@@ -0,0 +1,42 @@
+{
+    "apiVersion": "batch/v1",
+    "kind": "Job",
+    "metadata": {
+        "name": "{{arbimon-rfm-timestamp}}"
+    },
+    "spec": {
+        "ttlSecondsAfterFinished": 86400,
+        "template": {
+            "spec": {
+                "containers": [
+                    {
+                        "name": "arbimon-rfm-container",
+                        "image": "{{imagePath}}",
+                        "imagePullPolicy": "Always",
+                        "args": ["train_legacy"],
+                        "envFrom": [
+                            {
+                                "secretRef": {
+                                    "name": "arbimon-soundscapes-secret"
+                                }
+                            },
+                            {
+                                "configMapRef": {
+                                    "name": "arbimon-soundscapes-config"
+                                }
+                            }
```

Comment:
I would use different secret and config here. but you can keep using it to see if it is working with the rfm